pkgs/applications/networking/browsers/chromium/update.py

```diff
@@ -84,7 +98,10 @@ def get_channel_dependencies(channel):
             # the next one.
             continue
 
-        channel['deps'] = get_channel_dependencies(channel)
+        channel['deps'] = get_channel_dependencies(channel, channel_name)
+        if channel_name == 'stable':
+            channel['chromedriver'] = get_matching_chromedriver(channel['version'])
+            print(channel['chromedriver'])
```

Comment:
This print statement is probably confusing out of context; consider dropping it?

Reply:
Nice catch, thanks! I only had that for debugging purposes and missed it when reviewing the final diff.